GeosCore/aerosol_mod.F90

```diff
@@ -96,6 +96,9 @@ MODULE AEROSOL_MOD
   ! a new hygroscopic species) requires manual update of this mapping
   ! (ewl, 1/23/17)
   INTEGER  :: Map_NRHAER(5)
+
+  ! (hzhu Apr. 2024)
+  INTEGER  :: g 
```

Comment:
This should be a local variable where it is needed and not a top-of-module variable.  The reason being is that if it is at the top-of-module, it would be SAVEd throughout the run, which we wouldn't want.